### PR TITLE
[Feature] Introduce possibility to specify extravolumes for new runtimeScanner

### PR DESCRIPTION
**Pull request by dark-vex**
## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [X] PR only contains changes for one chart

### CODE REVIEW

**Comment by achandras** [comment]
@dark-vex do you mind adding these new fields to the readme? Otherwise LGTM!

**Comment by dark-vex** [comment]
@achandras I have updated the readme file, before merging this PR, let's wait for @ProvoK to approve the PR, he was running some regression test through Jenkins

**Review by ProvoK — APPROVED** [review]
LGTM thanks @dark-vex !

**Pull request merged**
